tests/test_resource.py

```diff
@@ -330,6 +317,15 @@ def test_options_request(self):
             ALLOW_ORIGIN_HEADER: ORIGIN_EXAMPLE,
         }
 
+    @patch.object(CORSResource, "resp_headers", return_value={})
+    def test_not_allow_oring_in_header(self, resp_headers_mock: MagicMock) -> None:
```

Comment:
typo: _"ori**gin**"_

Reply:
Please, describe it better. I think `CORSResource` allows having origin in headers but value can be unacceptable.